src/main/java/com/mitchej123/hodgepodge/core/commands/DebugCommand.java

```diff
@@ -0,0 +1,76 @@
+package com.mitchej123.hodgepodge.core.commands;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Stream;
+
+import net.minecraft.command.CommandBase;
+import net.minecraft.command.ICommandSender;
+import net.minecraft.util.ChatComponentText;
+
+import com.mitchej123.hodgepodge.core.util.AnchorAlarm;
+
+public class DebugCommand extends CommandBase {
+    @Override
+    public String getCommandName() {
+        return "hp";
+    }
+
+    @Override
+    public String getCommandUsage(ICommandSender sender) {
+        return "Usage: hp <subcommand>. Valid subcommands are: toggle, anchor.";
+    }
+    private void printHelp(ICommandSender sender) {
+        sender.addChatMessage(new ChatComponentText("Usage: hp <toggle|anchor>"));
+        sender.addChatMessage(new ChatComponentText("\"toggle anchordebug\" - toggles RC anchor debugging"));
+        sender.addChatMessage(new ChatComponentText("\"anchor list <player>\" - list RC anchors placed by the player (empty for current player)"));
```

Comment:
Could have been `"\"anchor list [player]\" - list RC anchors placed by the player (empty for current player)"`